package.json

```diff
@@ -100,5 +97,10 @@
   "resolutions": {
     "postcss-custom-properties": "^13.1.3",
     "react-refresh": "^0.11.0"
+  },
+  "devDependencies": {
+    "@commitlint/cli": "^18.4.3",
+    "@commitlint/config-conventional": "^18.4.3",
+    "husky": "^8.0.3"
```

Comment:
I noticed that on docker build husky cannot be found: https://dev.azure.com/City-of-Helsinki/tiedonohjaus/_build/results?buildId=139130&view=logs&j=9ddc01d6-dd91-59f0-4cac-fc0889c68c56&t=bebae947-0ef9-5a40-1eb7-e824ae25f4ff&l=170 Easy fix would be to move this from devDependencies to dependencies. 🤔